021/main.go

```diff
@@ -0,0 +1,18 @@
+// Daily Joel #21
+// Slices and zero values.
+package main
+
+import "fmt"
+
+func main() {
+	letters := make([]string, 100)
```

Comment:
Declaring and allocating a slice with the size of 100 indexes that we call letters